remo/events/forms.py

```diff
@@ -270,11 +270,15 @@ def clean(self):
             msg = 'Start date should come before end date.'
             self._errors['start_form'] = self.error_class([msg])
 
-        # Check that there is a cateogry selected
+        # Check that there is a category selected
         if not cdata.get('categories'):
             msg = 'You need to select one functional area for this event.'
             self._errors['categories'] = self.error_class([msg])
 
+        if self.instance.id is not None and cdata['start'] > now():
```

Comment:
I am not sure how this change removes the ask for event metrics. Basically it just blocks the ability to change an event date to the future.